manager/integration/tests/test_migration.py

```diff
@@ -406,7 +473,7 @@ def setup_migration_test(clients, volume_name, backing_image="", replica_cnt=REP
     Creates a new migratable volume then attaches it to the
     current node to write some test data on it.
     """
-    client = get_random_client(clients)
+    client = get_random_client(clients) # NOQA
```

Comment:
Do we need the `# NOQA` here?

Reply:
Hi @c3y1huang , 
In my environment if I did not add `# NOQA` here flake8 won't let me pass because I imported `client` at begging of script
```
manager/integration/tests/test_migration.py:476:5: F811 redefinition of unused 'client' from line 15
flake8 check failed
```

Reply:
I see, thanks for the explanation.